zeebe/engine/src/main/java/io/camunda/zeebe/engine/processing/resource/ResourceDeletionDeleteProcessor.java

```diff
@@ -142,6 +142,15 @@ public ProcessingError tryHandleError(
       rejectionWriter.appendRejection(command, RejectionType.NOT_FOUND, exception.getMessage());
       responseWriter.writeRejectionOnCommand(
           command, RejectionType.NOT_FOUND, exception.getMessage());
+
+      // TODO verify this fix. There is a risk attached to it as deployment creation and resource
```

Comment:
🔧 We can remove this comment as we now have a migration to set queue id for deployment create and resource delete distributions in the state.